src/Services/Masa.Auth.Service.Admin/Program.cs

```diff
@@ -142,7 +142,7 @@
 app.MigrateDbContext<AuthDbContext>((context, services) =>
 {
     var logger = services.GetRequiredService<ILogger<AuthDbContextSeed>>();
-    new AuthDbContextSeed().SeedAsync(context, logger).Wait();
+    //new AuthDbContextSeed().SeedAsync(context, logger).Wait();
```

Comment:
If you want comment code, you should provide a comment why.

Reply:
wrong operation, restored